app/models/micropost.rb

```diff
@@ -2,6 +2,7 @@ class Micropost < ApplicationRecord
   belongs_to :user
   has_one_attached :image
   default_scope ->{order created_at: :desc}
+  scope :by_authods, ->(following_ids, ids){where("user_id in (?) or user_id = ?", following_ids, ids)}
```

Comment:
nên dùng chung 1 mảng ids thôi em nhé 
```suggestion
  scope :by_authods, ->(ids){where user_id: ids}
```

Reply:
nếu vậy nó chỉ lấy những post có id là id của current_user thôi chớ chị ?